### PR TITLE
Order branch

### PR DESCRIPTION
**Pull request by marante**
- Created models for Customer, Clubmember, Adress, Country, Orders
- Added controller and dao for Orders
- Provided tests for Orders (to meet the reqs in the project instructions)

### CODE REVIEW

File: src/main/java/app/Application.java [hunk 1]
```diff
@@ -22,12 +25,18 @@ public static void main(String[] args) {
         Morphia morphia = new Morphia();
         final MongoClient client = new MongoClient();
         morphia.mapPackage("com.babayaga.beavercoffee.product"); // Can be called several times with diffrent packages if needed
+        morphia.mapPackage("com.babayaga.beavercoffee.order"); // Can be called several times with diffrent packages if needed
+        morphia.mapPackage("com.babayaga.beavercoffee.customer"); // Can be called several times with diffrent packages if needed
+        morphia.mapPackage("com.babayaga.beavercoffee.employee"); // Can be called several times with diffrent packages if needed
+        morphia.mapPackage("com.babayaga.beavercoffee.store"); // Can be called several times with diffrent packages if needed
```
Comment (Kalk88) [inline src/main/java/app/Application.java]: Add this line when store features gets implemented instead?

File: src/main/java/app/order/Order.java [hunk 2]
```diff
@@ -1,4 +1,33 @@
...
+        @Index(value = "timestamp", fields = @Field("timestamp"))
+)
 public class Order {
+    @Id
+    private ObjectId _id;
+    private String id;
+    private int timestamp;
+    private String discount;
+    @Embedded
+    private Customer customer;
```
Comment (Kalk88) [inline src/main/java/app/order/Order.java]: This should probably be an Order customer. Might break when we add the customer feature.

File: src/test/java/test_data/products_se.json [hunk 3]
```diff
@@ -1,3 +1,3 @@
 {
```
Comment (Kalk88) [inline src/test/java/test_data/products_se.json]: We can delete this file.

File: src/main/java/app/Application.java (hunk 1 above)
Comment (marante) [reply src/main/java/app/Application.java]: done.

File: src/test/java/test_data/products_se.json (hunk 3 above)
Comment (marante) [reply src/test/java/test_data/products_se.json]: done

**Comment by marante** [comment]
Please review again and approve

**Pull request closed without merging**